WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -1531,7 +1332,7 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
      * can we show the footer bar which contains the like & comment counts?
      */
     private fun canShowFooter(): Boolean {
-        return canShowLikeCount() || canShowCommentCount() || canShowBookmarkButton()
+        return canShowLikeCount() || canShowCommentCount() || canShowBookmarkButton() || canBeReblogged()
```

Comment:
AFAIK we never hide the actions in the post list - is this condition ever false? For what types of posts do we need to hide the footer?

Reply:
This condition was already present, I've not tested extensively if it can ever happen. I just didn't want to break existing logic by removing it.
That said I also think that it is fine to display footer with all actions disabled if it ever happens 🤷‍♀️.

Should I remove `canShowFooter` logic?

Reply:
Yeah, I understand why you didn't want to just remove it. At the same time, since we just worked on the post list logic I think no-one will have a better context than we do right now. So I'd just remove it.

Reply:
Removed in d6cc863